### PR TITLE
Alpine

### PR DESCRIPTION
**Pull request by gskerry**
Option to build and run via Docker image.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  && apk add pkgconfig \
+  && apk add build-base \
+  && apk add freetype-dev \
+  && apk add freetype \
+  && apk add p7zip ffmpeg \
+  && pip install virtualenv
+
+# CMD [ "bash", "env_setup.sh" ]
+
+#######
```
Comment (askerry) [inline Dockerfile:24]: did you mean to leave this stuff commented out here?

File: download.sh [hunk 2]
```diff
@@ -1,24 +0,0 @@
-#!/usr/bin/env bash
```
Comment (askerry) [inline download.sh]: we want to leave download.sh, setup.sh, etc. in place for the standard mac os setup right?

File: env_bootstrap/requirements.sh [hunk 3]
```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
```
Comment (askerry) [inline env_bootstrap/requirements.sh]: I'm a little confused about the env_bootstrap/requirements.sh vs. the env_bootstrap/requirements.txt does one make the other obsolete?

File: env_bootstrap/download.sh [hunk 4]
```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+echo "hit download script."
+
+ROOT=$(pwd)
+echo "check root: " $ROOT
+
+AUDIO_DIR="$ROOT/raw_data"
+
+# if [ ! -d raw_data ]
```
Comment (askerry) [inline env_bootstrap/download.sh]: did you mean to comment this out? was the data download not succeeding? (note: it's slow)

**Review by askerry — COMMENTED** [review]
woohoo!

**Comment by gskerry** [comment]
Will need to be a 2nd revision moving some of the RUN bootstrap scripting into the image build. Case-by-case modifications of scripts

File: env_bootstrap/download.sh (hunk 4 above)
Comment (gskerry) [reply env_bootstrap/download.sh]: No, that needs to be brought back in. Will update

File: env_bootstrap/requirements.sh (hunk 3 above)
Comment (gskerry) [reply env_bootstrap/requirements.sh]: Up to you I guess. Have not tested .sh version on osx. 
Would need to also keep .txt if you wanted to stick with that install approach.

Comment (askerry) [reply env_bootstrap/requirements.sh]: yeah I think leaving the requirements.txt at root makes sense, but maybe you can remove the env_bootstrap/requirements.txt version?

**Comment by askerry** [comment]
cool, I'm happy landing this onto master and continue to iterate from there!

**Review by askerry — APPROVED** [review]

File: env_bootstrap/requirements.sh (hunk 3 above)
Comment (gskerry) [reply env_bootstrap/requirements.sh]: Right. yep sounds good.

**Comment by gskerry** [comment]
Let me know if this repo organization looks like it will work for you.

**Review by askerry — APPROVED** [review]
LGTM!

**Pull request merged**